native/pom.xml

```diff
@@ -26,6 +26,11 @@
     <profiles>
         <profile>
             <id>mac</id>
+            <activation>
```

Comment:
I don't think we want to do this automatically; instead, have the user to explicitly enable this when needed.